core/contract/src/test/java/org/eclipse/dataspaceconnector/contract/negotiation/ProviderContractNegotiationManagerImplTest.java

```diff
@@ -1,6 +1,6 @@
 /*
 <<<<<<< HEAD
```

Comment:
This does not seem to be right.. committed some merge conflict lines?

Reply:
Fixed, NB those merged conflict lines were preexisting